*** testsBoth/A-061-061-M-ok.out	2023-10-19 17:31:46.706019000 +0100
--- testsBoth/A-061-061-M-ok.outhyp	2023-10-25 03:27:53.846161593 +0100
***************
*** 16,22 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 4
- Especifique a gama (startline;startcol:endline;endcol): 1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 16,21 ----
***************
*** 26,34 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1
! 1;1|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 25,35 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 1;1
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1
! 1;1|15
  Menu de Edição
  1 - Visualizar
  2 - Inserir
