*** testsBoth/A-064-064-M-ok.out	2023-10-19 17:31:46.828616000 +0100
--- testsBoth/A-064-064-M-ok.outhyp	2023-10-25 14:27:39.959873532 +0100
***************
*** 16,22 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 4
- Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 16,21 ----
***************
*** 26,31 ****
--- 25,32 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 5;1:5;3
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
  5;1|
