*** testsBoth/A-081-081-M-ok.out	2023-10-19 17:31:47.495943000 +0100
--- testsBoth/A-081-081-M-ok.outhyp	2023-10-26 03:15:15.788008248 +0100
***************
*** 40,46 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 6
- Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 40,45 ----
***************
*** 50,60 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
! 5;1|25
! 5;2|43
! 5;3|5=3;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 49,61 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 5;1:5;3
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
! 5;1|
! 5;2|
! 5;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
