*** testsBoth/A-090-090-M-ok.out	2023-10-19 17:31:47.692557000 +0100
--- testsBoth/A-090-090-M-ok.outhyp	2023-10-25 03:27:55.690149992 +0100
***************
*** 16,22 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 3
- Especifique a gama (startline;startcol:endline;endcol): 1;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 16,21 ----
***************
*** 26,33 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 6
- Especifique a gama (startline;startcol:endline;endcol): 1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 25,33 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 1;2
+ Número inválido!
  Escolha uma opção: 6
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 37,45 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1
! 1;1|49
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 37,47 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 1;1
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1
! 1;1|15
  Menu de Edição
  1 - Visualizar
  2 - Inserir
